project-base/app/src/FrontendApi/Resolver/Products/ProductsQuery.php

```diff
@@ -146,9 +156,19 @@ function ($offset, $limit) use ($argument, $productFilterData, $flag, $batchLoad
     }
 
     /**
-     * {@inheritdoc}
+     * @param \Overblog\GraphQLBundle\Definition\Argument $argument
      */
     public function productsQuery(Argument $argument)
+    {
+        throw new DeprecatedMethodException();
+    }
+
+    /**
+     * @param \Overblog\GraphQLBundle\Definition\Argument $argument
+     * @param \GraphQL\Type\Definition\ResolveInfo $info
+     * @return \App\FrontendApi\Model\Product\Connection\ProductExtendedConnection|\GraphQL\Executor\Promise\Promise
+     */
+    public function productsWithOverlyingEntityQuery(Argument $argument, ResolveInfo $info)
```

Comment:
```suggestion
    public function productsWithOverlayingEntityQuery(Argument $argument, ResolveInfo $info)
```

Reply:
Here I would guide the decision on this explanation:

_The terms "overlying" and "overlaying" are both related to the concept of something being on top of something else, but they are used in slightly different contexts.

    "Overlying": This term is often used in a more passive sense to describe something that is located above or on top of something else. It is often used in geological or medical contexts. For example, in geology, you might say that a layer of sandstone is "overlying" a layer of shale, meaning it is positioned above the shale in the strata. In medicine, you might refer to skin "overlying" a bone or muscle, meaning it covers it.

    "Overlaying": This term is often used in a more active sense to describe the act of placing something on top of something else. It is often used in the context of design, graphics, and technology. For example, in graphic design, you might "overlay" text on an image, meaning you actively place the text over the image. In technology, an "overlay" might refer to a user interface element that appears on top of other content on a screen.

So, in summary, "overlying" often describes a state of being or position, while "overlaying" often describes an action or process._

Because of that, I think overlying is more suitable